### PR TITLE
Update to security role in response to openssl

### PR DESCRIPTION
**Pull request by paulczar**
http://www.ubuntu.com/usn/usn-2537-1/

### CODE REVIEW

File: roles/security_errata/tasks/main.yml [hunk 1]
```diff
@@ -1,11 +1,13 @@
 ---
-- name: Patch CVE-2014-0160
+- name: update apt index
+  apt: update_cache=yes cache_valid_time=3600
+
+- name: update SSL
   apt: pkg={{ item }} state=latest
   with_items:
   - libssl1.0.0
   - openssl
```
Comment (j2sol) [inline roles/security_errata/tasks/main.yml]: Should we pin openssl version here so we ensure we're getting the updated version?

File: roles/security_errata/tasks/main.yml [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
-  notify: reload haproxy
-  tags: CVE-2014-0160
+  notify: restart haproxy
 
 - include: CVE-2015-0235.yml
-  tags: CVE-2015-0235
\ No newline at end of file
+  tags: CVE-2015-0235
```
Comment (j2sol) [inline roles/security_errata/tasks/main.yml]: Should we add more tags for the latest round?

Comment (paulczar) [inline roles/security_errata/tasks/main.yml]: nah, there's like 12 CVEs for it.   I think just the security tag should be enough.

File: roles/security_errata/tasks/main.yml (hunk 1 above)
Comment (paulczar) [inline roles/security_errata/tasks/main.yml]: latest should suffice.

Comment (j2sol) [inline roles/security_errata/tasks/main.yml]: Who decides what latest is though? I am unfamiliar with how apt works, if one gets mirrored to an out of date mirror, would it see the old openssl as "latest" and think all is good?

**Comment by j2sol** [comment]
Hrm, I wonder why jenkins didn't pick this up.

**Pull request merged**